snap-jp2-reader/src/main/java/org/esa/snap/jp2/reader/metadata/Jp2XmlMetadata.java

```diff
@@ -214,4 +225,15 @@ public List<Point2D> getPolygonPositions() {
         }
         return null;
     }
+
+    /**
+     * SNAP-3459 (https://senbox.atlassian.net/browse/SNAP-3459)
+     * Do not rely on EPSG and check the actual axis name order
+     * @return true is the axis order is reversed (e.g. [y,x] instead of [x,y])
+     */
+    private boolean isReversedAxisOrder() {
+        String[] axisName = getAttributeValues(JP2ProductReaderConstants.TAG_AXIS_NAME);
+        return (axisName[0].contains("y") || axisName[0].contains("lat")) ? true : false;
```

Comment:
can be simplified to:  
`return axisName[0].contains("y") || axisName[0].contains("lat");`